pandas/core/resample.py

```diff
@@ -519,21 +519,55 @@ def nearest(self, limit=None):
 
     def backfill(self, limit=None):
         """
-        Backward fill the values
+        Backward fill the values.
+
+        Resample datetimelike data and fill backwards missing values if any.
 
         Parameters
         ----------
         limit : integer, optional
-            limit of how many values to fill
+            Limit of how many values to fill.
 
         Returns
         -------
         an upsampled Series
```

Comment:
I've seen some examples on https://python-sprints.github.io/pandas/guide/pandas_docstring.html, which would take along the lines of: 

Returns
-------
Series
         An upsampled Series with backward filled NA values